react/game/src/redux/auth/constants.js

```diff
@@ -1,2 +1,6 @@
-export const TARGET_APP_LOADING = 'appLoading';
-export const TARGET = 'authInfo';
+const targets = {
+  TARGET: 'authInfo',
```

Comment:
This is better! But keep in mind that it'd be easier to understand your code if the names you use are a bit more descriptive. :)

Reply:
solved, sorry i had not understood the changes that you had requested.